renderlib/Logging.cpp

```diff
@@ -68,3 +76,38 @@ Logging::Enable(bool enabled)
 {
   boost::log::core::get()->set_logging_enabled(enabled);
 }
+
+boost::filesystem::path
+getLogPath()
+{
+#if defined(WIN32) || defined(_WIN32) || defined(__WIN32__) || defined(__NT__)
+  const char* rootdir = getenv("LOCALAPPDATA");
+  return boost::filesystem::path(rootdir) / "AllenInstitute" / "agave";
+#elif __APPLE__
+  const char* rootdir = getenv("HOME");
+  if (!rootdir) {
+    struct passwd* pwd = getpwuid(getuid());
+    if (pwd)
+      rootdir = pwd->pw_dir;
+  }
+  return boost::filesystem::path(rootdir) / "Library" / "Logs" / "AllenInstitute" / "agave";

```

Comment:
TIL python Pathlib probably copied this C++ API with `/` for continued path "division"

Reply:
Also, fun fact:  C++17 standard library has a `filesystem` module that is almost a perfect copy of the boost::filesystem API.    But there's still no cross platform universal "get current user home directory" function.

Reply:
`Path("~").expanduser()` is v nice

Reply:
Not sure if "~" has a Windows implementation